### ISSUE TITLE
Edit Task UI does not populate task's start date and end date time fields

### ISSUE THREAD

**Issue by mgmccarthy** [HTBox/allReady#1124]
editing an existing task:
![image](https://cloud.githubusercontent.com/assets/2428993/17563384/2c7dda82-5efd-11e6-93b4-26333f19d9ba.png)

although the model is returning the start date and end date time:
![image](https://cloud.githubusercontent.com/assets/2428993/17563422/5744ad0e-5efd-11e6-872f-1357faceb54b.png)

The UI is not showing it when it loads:
![image](https://cloud.githubusercontent.com/assets/2428993/17563438/673e5976-5efd-11e6-828e-75519722e88b.png)

Might be related to Issue #1107?

**Comment by mgmccarthy** [comment]
quick update on this issue. 

It appears as if the `class="form-control datetimepicker"` is causing this issue for start and end date:
```
<input asp-for="StartDateTime" class="form-control datetimepicker" />
<input asp-for="EndDateTime" class="form-control datetimepicker" />
```

when removing the class from each of the controls:
```
<input asp-for="@Model.StartDateTime" />
<input asp-for="@Model.EndDateTime" />
```

I can see the correct start and end date time values being loaded into the start and end date fields (even though it's not pretty, the data is there)
![image](https://cloud.githubusercontent.com/assets/2428993/17564715/6cc50070-5f02-11e6-932e-07fbe2dffe61.png)

does anyone have any idea if the existing `class="form-control datetimepicker"` has undergone changes, updated a nuget package, or even worked correctly since the RTM update?

**Comment by stevejgordon** [comment]
@mgmccarthy I think this broke since RTM as that was the first time I noticed an issue. @VishalMadhvani has a PR (#1025) in which I think might address this.

**Comment by VishalMadhvani** [comment]
I can't see how its possible for the RTM update to break this.
My understanding of the issue is the datepicker plugin being used, clears the date when its initialised.
PR #1025 fixes that but I'm running into trouble with localisation.

Happy for someone to pick up the pr and crack on as I've not got much time to look into it.

**Comment by stevejgordon** [comment]
@VishalMadhvani Purely guessing, but perhaps some changes in the razor processing somewhere? I can't specially think of what that might be though. I could be wrong and perhaps it broke prior to RTM for some other reason.

**Comment by mgmccarthy** [comment]
@stevejgordon  @VishalMadhvani 

What I'm seeing when the page loads sometimes is the start and end date do get populated ,but only for a split second before they're both cleared out. Sometimes I won't see it at all. It's almost like there some type of page refresh/DOM refresh that happening that is clearing them out.

I tried to comment out/remove all the knockout code from the Task create/edit page, and that did not fix it... so it's definitely not a knockout issue.

Either way, I didn't realize there was another Issue/PR open that addresses this issue. Do you want me to close this issue?

If not, I'll link this issue to PR #1025 issue. I'm also thinking Issue #1107 will be fixed as well once the PR is merged.

**Comment by mgmccarthy** [comment]
this Issue was fixed when PR #1025 was merged. It can be closed.
